docs/api/admin/tags-api.md

```diff
@@ -0,0 +1,174 @@
+---
+id: tags
+title: /api/admin/tags
+---
+
+> In order to access the admin API endpoints you need to identify yourself. If you are using the `insecure` authentication method, you may use [basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) to identify yourself.
+
+### Create a new tag
+
+`POST https://unleash.host.com/api/admin/tags`
+
+Creates a new tag without connecting it to any other object, can be helpful to build an autocomplete list.
```

Comment:
I wonder if tags also should expose a "autocomplete" endpoint. E.g 
`POST https://unleash.host.com/api/admin/tags/autocomplete?type=simple&value=a` 

Should return all tag values for a tag-type=simple and value starts with "a" (ignoring case).

Reply:
As a POST or as a GET?